client/src/services/return-of-service.js

```diff
@@ -61,3 +61,31 @@ export const getAllSites = async () => {
     throw new Error('Failed to fetch all sites');
   }
 };
+
+export const onRosUpdate = async (participantId, newValues) => {
```

Comment:
```suggestion
export const updateRosStatus = async (participantId, newValues) => {
```

This naming is too coupled to the calling component, it should describe what it's doing, not what's calling it